scripts/linters/python_linter.py

```diff
@@ -42,12 +42,33 @@
 # pylint: disable=wrong-import-order
 # pylint: disable=wrong-import-position
 from pylint import lint  # isort:skip
+from pylint.reporters import text# isort:skip
 import isort  # isort:skip
 import pycodestyle # isort:skip
 # pylint: enable=wrong-import-order
 # pylint: enable=wrong-import-position
 
 
+class PylintReport(python_utils.OBJECT):
+    """Output stream for pylint."""
+
+    def __init__(self):
+        """Constructs a PylintReport object."""
+        self.messages = []
+
+    def write(self, message):
+        """Write method for pylint.
```

Comment:
This docstring is a bit unhelpful. Maybe explain what the message is being written to (a docstring would typically say something like "Writes the given message to ...").